client/src/Components/Header/Header.js

```diff
@@ -1,24 +1,16 @@
 import React from "react";
 import './Header.css';
-import { Link, Route, Switch } from "react-router-dom";
-
 
 
 const Header = () => {
   return (
     <>
-      <Switch>
-        <Route exact path='/'></Route>
-        <Route path='*'>
-          <header>
-            <nav>
-              <Link to='/' className='logo'>BEAM</Link>
-            </nav>
-          </header>
-        </Route>
-      </Switch>
-    </>
-    
+      <header>
+        <nav>
+          <p to='/' className='logo'>BEAM</p>
```

Comment:
i don't think we need to `to='/'` bit now

Reply:
Beautiful!!!